public/404.html

```diff
@@ -1,67 +1,85 @@
 <!DOCTYPE html>
-<html>
+<html class="govuk-template">
```

Comment:
html tag needs a lang attribute so that it will be pronounced correctly by assistive technologies such as screen readers.

`<html lang="en">`